notifier/selfstate/checkable/checkabletypes.go

```diff
@@ -0,0 +1,84 @@
+package checkable
```

Comment:
Don't you think that `checkabletypes.go` for file name is little bit redundant if we are talking about file in package `checkable`? I think `types.go` will be enough for file name.

Reply:
Renamed)